homework/index.js

```diff
@@ -30,18 +30,100 @@
     return elem;
   }
 
-  function main(url) {
+  function renderRepositoryDescription(leftContainer, repository) {
+    const parentTable = createAndAppend('parentTable', leftContainer, { id: 'parentTable' });
+    const nameTable = createAndAppend('tr', parentTable, { class: 'descriptions' });
+
+    createAndAppend('td', nameTable, { class: 'titlesName', text: 'Repository:' });
+    createAndAppend('td', nameTable, { class: 'titlesValue', text: repository.name });
```

Comment:
I noticed one more thing: the repository name is supposed to a clickable link to the relevant repository (see homework description).

Reply:
also fixed, will push everything when I can fix the bug.. just don't know where is it yet. @remarcmij